primitiv/core/operator_impl.cc

```diff
@@ -414,6 +417,8 @@ FORWARD(Pow) { *y[0] = functions::pow(*x[0], *x[1]); }
 FORWARD(Transpose) { *y[0] = functions::transpose(*x[0]); }
 FORWARD(MatrixMultiply) { *y[0] = functions::matmul(*x[0], *x[1]); }
 
+FORWARD(Flip) { *y[0] = functions::sum(*x[0], dim_); }
```

Comment:
This looks just a bug, and the test for this routine (in operator_impl_test.cc) is missing.